api_docs/server_doc.py

```diff
@@ -0,0 +1,350 @@
+                },
+                {
+                    "@type": "SupportedProperty",
+                    "property": "http://schema.org/fuelCapacity",
+                    "readable": "true",
+                    "required": "false",
+                    "title": "Battery",
+                    "writeable": "true"
+                },
+                {
+                    "@type": "SupportedProperty",
+                    "property": "http://schema.org/device",
+                    "readable": "true",
+                    "required": "false",
+                    "title": "Sensor",
+                    "writeable": "false"
+                },
+                {
+                    "@type": "SupportedProperty",
+                    "property": "http://schema.org/model",
+                    "readable": "true",
+                    "required": "false",
+                    "title": "Model",
+                    "writeable": "false"
+                },
+                {
+                    "@type": "SupportedProperty",
+                    "property": "https://schema.org/status",
+                    "readable": "true",
+                    "required": "false",
+                    "title": "Status",
+                    "writeable": "true"
+                }
+            ],
+            "title": "LogEntry"
+        },
+        {
+            "@context": "http://hydrus.com/context/Area.jsonld",
+            "@id": "http://hydrus.com/serverapi/vocab/Area",
+            "@type": "Class",
+            "description": "Class for Area of Interest of the server",
+            "supportedOperation": [
+                {
+                    "@id": "/serverapi/update_area",
+                    "@type": "Operation",
+                    "expects": "http://hydrus.com/Area",
+                    "method": "PUT",
+                    "possibleStatus": [
+                        {
+                            "code": 200,
+                            "description": "Area of interest changed"
+                        }
+                    ],
+                    "returns": "null",
+                    "title": "UpdateArea"
+                }
+            ],
+            "supportedProperty": [
+                {
+                    "@type": "SupportedProperty",
+                    "property": "http://schema.org/geo",
+                    "readable": "true",
+                    "required": "true",
+                    "title": "TopLeft",
+                    "writeable": "true"
+                },
+                {
+                    "@type": "SupportedProperty",
+                    "property": "http://schema.org/geo",
+                    "readable": "true",
+                    "required": "true",
+                    "title": "BottomRight",
+                    "writeable": "true"
+                }
+            ],
+            "title": "Area"
+        }
+    ],
+    "title": "API Doc for the server side API"
+}
```

Comment:
We'll need to add more endpoints from the design doc, rest everything is fine! 👍